README.md

```diff
@@ -230,6 +230,10 @@ __Parameters__:
 
 - Download requests is being cached by native implementation. To avoid caching, pass `if-Modified-Since` header to download method.
 
+### Windows Quirks
+
+- Usage of [BackgroundDownloader](https://msdn.microsoft.com/en-us/library/windows/apps/windows.networking.backgroundtransfer.backgrounddownloader.aspx) entails the latency issues on Windows Phone. You can use XHR or [HttpClient](https://msdn.microsoft.com/en-us/library/windows/apps/windows.web.http.httpclient.aspx) as a quicker alternative for small downloads.
```

Comment:
Moved quirk out from `download` method section as it affects uploads as well; added some details on the latency issue.